sdk/iOS/src/MSCoreDataStore.m

```diff
@@ -65,6 +65,9 @@ -(id) getRecordForTable:(NSString *)table itemId:(NSString *)itemId asDictionary
     
     if (item && asDictionary) {
         
+        //Someone, please, test this. Solves the warning, but I don't know it it will work as intended.
+//        NSDictionary *result = [item dictionaryWithValuesForKeys:[[[item entity] attributesByName] allKeys]];
+        
```

Comment:
I am not sure that the resulting dictionary will contain the same keys and values.


> Em 21/09/2015, à(s) 14:39, Shrirang Shirodkar <notifications@github.com> escreveu:
> 
> In sdk/iOS/src/MSCoreDataStore.m <https://github.com/Azure/azure-mobile-services/pull/797#discussion_r40000608>:
> 
> > @@ -65,6 +65,9 @@ -(id) getRecordForTable:(NSString *)table itemId:(NSString *)itemId asDictionary
> >      
> >      if (item && asDictionary) {
> >          
> > +        //Someone, please, test this. Solves the warning, but I don't know it it will work as intended.
> > +//        NSDictionary *result = [item dictionaryWithValuesForKeys:[[[item entity] attributesByName] allKeys]];
> > +        
> This change looks good. The next line in the if-block ([NSDictionary dictionaryWithDictionary:result]) is not needed after this change as 'result' is not NSKnownKeysDictionary.
> This change will also take care of issue #779 <https://github.com/Azure/azure-mobile-services/issues/779>.
> 
> I am curious to know what warning you were seeing.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Azure/azure-mobile-services/pull/797/files#r40000608>.
>